user-guide/UserGuide.md

```diff
@@ -0,0 +1,237 @@
+**Figure 3.** Write the facts that will be included in the lesson.
+
+
+## The Question Document
+
+### Naming convention for a question document
+
+When saving a question document file, capitalize every word of the filename including conjunctions. The filename is composed of the following:
+
+1.	the name of the lesson
+2.	the suffix ".q"
+3.	the file extension ".md"
+
+![How to name a question document](https://lh4.googleusercontent.com/XmuBgnM4mhPd7zYlS0Dx_O9wTX0bkCBfk2n7jURwKOMn9HIXSfN_zsqXNY0cqzQ6UDAJWSPXyJwv1pE=w1920-h950-rw)
+
+&nbsp;&nbsp;&nbsp;**Figure 4.** When naming a question document, capitalize every word including conjunctions.
+
+### Annotations
+
+Annotations denote how a group of text will be rendered by the reader. The annotation for headers are as follows:
+#### One number sign #
+
+Use one number sign # to denote the title of the lesson.
+
+#### Three number signs ###
+
+Use three number signs ### to denote a specific part of a lesson.
+
+#### Three slash signs /// 	
+
+Use three slash signs /// to denote the type of a question or render a code editor. 
+
+![Three slash signs](https://lh6.googleusercontent.com/4E0N5-oE5VB3xFmbi3yjKyEvTiMmYWtt2PE75ainRKRsledtUuS1cXBDntVRWyrCpgyqK8myOk49On0=w1920-h950)
+
+**Figure 5.** In the first instance, the slashes render a code editor. 
+              In the second instance, the slashes denote the type of question.
+
+### How to Annotate an Answer Key
+
+Before each question, create a two-part annotation that indicates:
+* the type of question; and
+* the correct answer. 
+
+This is called the answer key. The answer key allows the reader to locate the correct answer. 
+
+To annotate an answer, follow these steps:
+1.	Use three slash signs **///** to denote that the text that follows indicates the type of question and the answer key.
+2.	Write **type=**`type of question`, followed by a comma **,**. 
+    </br>*See **Types of Questions** below to know how to annotate for different types of questions.*
+3.	Then, write **answer=**[`answerwithinthebrackets`].
+4.	Within the square brackets, indicate the correct answer. For single selection and multiple selection, the correct answer is indicated by a number: **1** being the first answer in the selection, **2** being the second, and so forth.
+
+For code response answers, indicate the path of the file that contains the correct answer.
+
+![Include the path for coder response](https://lh4.googleusercontent.com/tEOJrxGXeXL_-IjQvorqpWW2PsxispGYln3UFVOYpm7wIMbXnibXZekbaaAWQ3cGL_ng25uNJTTJQrI=w1920-h950-rw) 
+**Figure 6.** For Code Response questions, indicate the path of the file that contains the correct answer.
+
+### Types of Questions
+
+There are five annotations that indicate the type of question. 
+
+**MS**. **MS** stands for multiple selection. This type of question requires two or more selections. Use this annotation for a multiple selection question: **///type=MS**.
```

Comment:
A space after `///` is required like  `/// type=MS`. This applies to all types of questions. Thanks.